src/plugins/europeana/search.js

```diff
@@ -28,22 +28,22 @@ export const unquotableFacets = [
 ];
 
 /**
- * Construct a range query from two values, if keys are omitted they will default to '*'
- * @param {Object[]} values An object containing 'start' and 'end' values
- * @return {string} The range as a value that can be used by the API
- */
+  * Construct a range query from two values, if keys are omitted they will default to '*'
```

Comment:
Undo the JSdoc indentation changes throughout this file, which are inconsistent with that used in e.g. the new component documentation.